mars/deploy/local/core.py

```diff
@@ -300,6 +304,10 @@ def new_cluster(address='0.0.0.0', web=False, n_process=None, shared_memory=None
     web_process = None
     if web_endpoint:
         web_process = _start_web_process(endpoint, web_endpoint)
+        print('Web endpoint started at http://%s' % web_endpoint, file=sys.stderr)
```

Comment:
Is it better to use logger instead of print?

Reply:
The default log level is warning, but it is strange to display this message as warning.